### PR TITLE
Fix tags in metadata files for migrated community connectors 

### PR DESCRIPTION
**Pull request by DanyloGL**
Some of migrated to low-code community connectors had no updated language tags in metadata file.
List of connectors:

- source-braintree
- source-everhour
- source-klaus-api
- source-plaid
- source-unleash

## What
Language tags of connectors were fixed
## How
Was added language tag `low-code` in metadata files of listed above connectors.

## 🚨 User Impact 🚨
No breaking changes

### CODE REVIEW

File: airbyte-integrations/connectors/source-braintree/metadata.yaml [hunk 1]
```diff
@@ -24,5 +24,6 @@ data:
   releaseStage: alpha
   supportLevel: community
   tags:
+    - language:low-code
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-braintree/metadata.yaml:27]: we need leave only one tag 

language:low-code or language:python should be presented in metadata file

**Review by lazebnyi — CHANGES_REQUESTED** [review]

**Review by lazebnyi — APPROVED** [review]
lgtm

**Comment by octavia-approvington** [comment]
It's time <br/> fine lets go

**Review by octavia-approvington — APPROVED** [review]

**Pull request merged**